types/static.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
What is this declaration doing?

Reply:
Ah good catch. An artifact of some typing experimentation. Removed.